gnovm/pkg/gnolang/uverse.go

```diff
@@ -9,6 +9,13 @@ import (
 
 	bm "github.com/gnolang/gno/gnovm/pkg/benchops"
 	"github.com/gnolang/gno/tm2/pkg/crypto"
+	"github.com/gnolang/gno/tm2/pkg/overflow"
+	"github.com/gnolang/gno/tm2/pkg/store/types"
+)
+
+const (
+	NativeCPUUversePrintInit    = 1800
+	NativeCPUUversePrintPerChar = 1
```

Comment:
can you please point out where these values are calculated?

Reply:
updated the description to add the op code and native benchmark